setup.cfg

```diff
@@ -35,6 +35,7 @@ openstack.esiclient.v1 =
     esi_node_network_detach = esiclient.v1.node_network:Detach
     esi_node_network_list = esiclient.v1.node_network:List
     esi_node_volume_attach = esiclient.v1.node_volume:Attach
+    esi_mdc_baremetal_node_list = esiclient.v1.node_baremetal:MDCBaremetalNodeList
```

Comment:
These are generally in alphabetical order, so esi_mdc would go above all the esi_node entries.